src/django_idom/templatetags/idom.py

```diff
@@ -0,0 +1,38 @@
+import json
+from urllib.parse import urlencode
+from uuid import uuid4
+
+from django import template
+
+from django_idom.app_settings import (
+    IDOM_WEB_MODULES_URL,
+    IDOM_WEBSOCKET_URL,
+    TEMPLATE_FILE_PATHS,
+)
+
+from ..app_components import has_component
+
+
+register = template.Library()
+
+
+# Template tag that renders the IDOM scripts
+@register.inclusion_tag(TEMPLATE_FILE_PATHS["head_content"])
+def idom_head():
+    pass
+
+
+@register.inclusion_tag(TEMPLATE_FILE_PATHS["view"])
```

Comment:
Looks like we're providing `register.inclusion_tag` a full file path to the bare _development_ template file. This will result in unusable code for production environments where `DEBUG` is turned off and the user uses `python manage.py collectstatic`.

You should only pass in a sub-path paths that of what comes after the `templates/` dir.

To mimic this we need to `lstrip()` anything before `idom/templates/` (inclusive).

Example:
Original path: `/home/my_app/idom/templates/idom/views.html`
New path: `idom/views.html`

Reply:
For some reason I think I was having issues with using the string in CI. Will try and revert back to using a string instead to be sure.

Reply:
If you're having issues with sub paths then it's possible your template related settings in settings.py are misconfigured.